### PR TITLE
Update `Installing OctoScreen with a 3.5" 480x320 TFT screen` guide

### PR DESCRIPTION
**Pull request by johslarsen**
A lot have happened in the years since this project, the [display configuration](https://github.com/waveshare/LCD-show) and the [display driver](https://github.com/ssvb/xf86-video-fbturbo) all became unmaintained and outdated. Long story short, the old guide does not work anymore, so I wrote up what I did successfully to install OctoScreen on my printer.

This is mostly a summary that I would like to have available the next time I need to reinstall my device, but others might also find it beneficial. Seems to work a lot better than the old guide at least. The old guide was in the wiki, but as far as I know there is no easy way to make merge requests there (and having an open merge request might be a place people actually look), so I copied that information into the repository and just linked from the project README.md there instead of to the wiki.

### CODE REVIEW

File: docs/Installing-OctoScreen-with-a-3.5"-480x320-TFT-screen.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+```sh
+git clone https://github.com/waveshare/LCD-show.git
+cd LCD-show
+
+# but this is unmaintained and outdated, so we have to do some damage control (i.e. replace relevant lines with noops):
+sed -i 's/.*99-fbturbo.conf.*/true/' LCD35-show      # because that driver is unmaintained and incompatible with todays Xorg, default works fine though
+sed -i 's/.*reboot.*/true/' LCD35-show               # because we have more to do
+
+sudo ./LCD-35-show  # NOTE: default is screen orientation with power plug on the bottom, e.g. `sudo ./LCD-35-show 180` to have it reversed
```
Comment (ammgws) [inline docs/Installing-OctoScreen-with-a-3.5"-480x320-TFT-screen.md]: `s/LCD-35-show/LCD35-show`

Comment (johslarsen) [reply docs/Installing-OctoScreen-with-a-3.5"-480x320-TFT-screen.md]: thanks, fixed

**Comment by JeffB42** [comment]
@johslarsen and @ammgws - thanks for contributing to this project! (it really helps, as I don't have much time to maintain the docs, which always seem to be in a state of flux)

_"A lot have happened in the years since this project, the [display configuration](https://github.com/waveshare/LCD-show) and the [display driver](https://github.com/ssvb/xf86-video-fbturbo) all became unmaintained and outdated. Long story short, the old guide does not work anymore, so I wrote up what I did successfully to install OctoScreen on my printer."_
This is excellent!  Thanks!


_"The old guide was in the wiki, but as far as I know there is no easy way to make merge requests there"_
Yea, this is an issue.  As far as I know, there isn't an easy way to submit changes or PRs for wiki pages.

For these changes, I'm thinking what I will do is copy/paste the contents of this PR into the (existing) wiki page, and then close this PR.

Thoughts?

Otherwise, if we keep it as a doc in the repo, 1) this PR should target the `2.8.0-dev` branch and not `master`, and 2) I should delete the page from the wiki, to avoid confusion of having two different docs, in two different locations.

Oh, also, I'm thinking about publishing a beta of 2.8.0 soon.
(and I promise, I'll try to get 2.8.0 released before the end of Summer!)

**Comment by johslarsen** [comment]
> For these changes, I'm thinking what I will do is copy/paste the contents of this PR into the (existing) wiki page, and then close this PR.
> 
> Thoughts?

Fine by me. I thought this was unmaintained, so just wanted to make sure the updates were public.

> Oh, also, I'm thinking about publishing a beta of 2.8.0 soon. (and I promise, I'll try to get 2.8.0 released before the end of Summer!)

Great, I will make sure to test it out the beta.

**Comment by JeffB42** [comment]
@johslarsen OK, I updated https://github.com/Z-Bolt/OctoScreen/wiki/Installing-OctoScreen-with-a-3.5%22-480x320-TFT-screen.  If you see any glaring errors with my copy/paste, please let me know, and again, thanks for the PR and contributing.

Jeff

**Pull request closed without merging**